recipe/meta.yaml

```
@@ -41,14 +41,28 @@ test:
     - pandas
   commands:
     - pip check
+    # Skip test suite on PyPy as it segfaults there
+    # xref: https://github.com/conda-forge/pandas-feedstock/issues/148
+    {% set extra = "extra_args=['-m not clipboard and not single_cpu', '--skip-slow', '--skip-network', '--skip-db', '-n=2']" %}
+    - python -c "import pandas; pandas.test({{ extra }})"  # [python_impl == "cpython"]
   requires:
     - pip
+    - pytest >=6.0
+    - pytest-asyncio >=0.17
+    - pytest-xdist >=1.31
+    - pytest-cov
+    - boto3
+    - hypothesis
+    - psutil
 
 about:
   home: http://pandas.pydata.org
   license: BSD-3-Clause
   license_file: LICENSE
-  summary: High-performance, easy-to-use data structures and data analysis tools.
+  summary: >-
+    Powerful data structures for data analysis, time series, and statistics
+  doc_url: https://pandas.pydata.org/pandas-docs/stable
```

```suggestion
  doc_url: https://pandas.pydata.org/docs/
```

(that's the default link we use nowadays (eg linked from the website), seems we need to update setup.cfg then as well)

---

FWIW this link came from [the `setup.cfg`]( https://github.com/pandas-dev/pandas/blob/8dab54d6573f7186ff0c3b6364d5e4dd635ff3e7/setup.cfg#L29 ). It also shows up in [the newer `pyproject.toml`]( https://github.com/pandas-dev/pandas/blob/bf5ee72d5b81962db91fa12f637b4369d3e30f77/pyproject.toml#L51 ). So maybe worth updating those too

---

Filed upstream PRs to update the doc URLs:

* https://github.com/pandas-dev/pandas/pull/50278
* https://github.com/pandas-dev/pandas/pull/50279

---

The upstream PRs have been merged and this change has been incorporated